src/linagora.esn.contact/app/addressbook/addressbook-shared-configuration/addressbook-shared-configuration.controller.js

```diff
@@ -1,7 +1,6 @@
 const _ = require('lodash');
 
-require('../addressbook.service.js');
-require('../addressbook.constants.js');
+require('../../../../esn.contact.libs/app/addressBook/addressbook.service.js');
```

Comment:
No need this. Instead, you need to require `app.module` file of `esn.contact.libs` in `linagora.esn.contact` app.module file

Reply:
when i remove this require , it breaks the SPA

Reply:
You need to check if the app.module file of `esn.inbox.libs` is required correctly

Reply:
How about this one?